endpoints-support.md

```diff
@@ -316,11 +345,15 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/power-devices/{id}/powerState</sub>                                           | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/power-devices/{id}/powerState</sub>                                           | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/power-devices/{id}/refreshState</sub>                                         | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/power-devices/{id}/synchronous</sub>                                          | DELETE   |  :x:  | :x:   | :x:   |
+|<sub>/rest/power-devices/{id}/synchronous</sub>                                          | DELETE   |  :heavy_multiplication_x:      | :heavy_multiplication_x:       | :heavy_multiplication_x:       |
 |<sub>/rest/power-devices/{id}/uidState</sub>                                             | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/power-devices/{id}/uidState</sub>                                             | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/power-devices/{id}/utilization</sub>                                          | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
-|     **Rack**                                                                                                                                     |
+|     **Providers**                                                                                                                                     |
+|<sub>/rest/fc-sans/providers</sub>                                                       | GET      | :heavy_multiplication_x:       | :heavy_multiplication_x:       |   :heavy_multiplication_x:  |
+|<sub>/rest/fc-sans/providers/{id}</sub>                                                  | GET      | :heavy_minus_sign:   | :heavy_multiplication_x:       |   :heavy_multiplication_x:  |
+|<sub>/rest/fc-sans/providers/{id}/device-managers</sub>                                  | POST     | :heavy_minus_sign:   | :heavy_multiplication_x:       |   :white_check_mark:  |
```

Comment:
Shouldn't it be ✅ for API300?

Reply:
Fixed and removed "out of scope" from the first get to add this endpoint in the backlog.